usaspending_api/etl/award_helpers.py

```diff
@@ -145,11 +129,41 @@ def update_awards(award_tuple: Optional[tuple] = None) -> int:
         "  WHERE e.award_id = a.id "
     )
 
-    sql_prune = prune_empty_awards_cte
-
     sql_update = _sql_update.format(earliest_transaction_cte, latest_transaction_cte, aggregate_transaction_cte)
     # We don't need to worry about this double counting awards, because if it's deleted in the first step it can't be updated!
-    return execute_database_statement(sql_prune, [award_tuple]) + execute_database_statement(sql_update, values)
+    return prune_empty_awards(award_tuple) + execute_database_statement(sql_update, values)
+
+
+def prune_empty_awards(award_tuple: Optional[tuple] = None) -> int:
+    _find_empty_awards_sql = str(
+        "SELECT a.id "
+        "FROM awards a "
+        "LEFT JOIN transaction_normalized t "
+        "ON t.award_id = a.id "
+        "WHERE t is null {}"
+    ).format(" AND a.id IN %s " if award_tuple else "")
+
+    _modify_subawards_sql = (
+        "UPDATE subaward " "SET award_id = null " "WHERE award_id IN ({});".format(_find_empty_awards_sql)
```

Comment:
These strings in line 147 can be combined to a single string, same with line 161